src/pages/ReportsUpload/UploadForm.tsx

```diff
@@ -52,7 +52,7 @@ class UploadForm extends React.Component<UploadFormProps, { }> {
             setTimeout(() => {
                 this.props.setFieldValue('file', file);
                 if (!this.props.values.reportName) {
-                    const fileNameWithoutExtension = file.name.replace(/\.[^/.]+$/, "");
+                    const fileNameWithoutExtension = file.name.replace(/(\.[^/.]+)+$/, "");
```

Comment:
Is there any test covering the change with different use cases ?

Reply:
@jonathanvila I've just added a new commit which is including a couple of tests to cover this change